data-api.js

```diff
@@ -0,0 +1,123 @@
+const sqlstring = require("./sqlstring");
+const dataApiClient = require("data-api-client");
+const Bluebird = require("bluebird");
+const DataAPITransaction = require("./data-api-transaction");
+
+// Call mysql client to setup knex, this set as this function
+function dataAPI(ClientRDSDataAPI, Client, dialect) {
+  Object.setPrototypeOf(ClientRDSDataAPI.prototype, Client.prototype);
+
+  // Add/change prototype functions and properties
+  Object.assign(ClientRDSDataAPI.prototype, {
+    driverName: "rds-data",
+
+    _driver() {
+      // Setup dataApiClient
+      return dataApiClient({
+        secretArn: this.config.connection.secretArn,
+        resourceArn: this.config.connection.resourceArn,
+        database: this.config.connection.database,
+        region: this.config.connection.region
+      });
```

Comment:
Would it be good to have option to pass all possible [parameters](https://github.com/jeremydaly/data-api-client#configuration-options) to data-api-client? This would also make possible local development against local-data-api

Reply:
So, more or less just pass this.config.connection as parameters? Think that would work. Should we add som check here to ensure that the required parameters are available?

Reply:
Went ahead and changed it to pass the config.connection. Also changed to use the newly published version of the data-api-client as the region fix seems to be in now. Haven't tested it thoroughly yet..

Reply:
Did a quick test with local setup and this seems to work fine :)